src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    margin-right: 0;
+  }
+
+  &:nth-child(1) > .stars__star:nth-child(-n + 4) {
+    background-image: url("../images/star-active.svg");
+    margin-right: 4px;
+  }
```

Comment:
you can also try to take it work with @each - SCSS loop 😉

Reply:
I did it with @for ;)